std/testing/asserts_test.ts

```diff
@@ -159,6 +160,19 @@ Deno.test("testingNotEquals", function (): void {
   assertEquals(didThrow, true);
 });
 
+Deno.test("testingAssertAnything", function (): void {
+  assertAnything("Denosaurus");
+  let didThrow;
+  try {
```

Comment:
We should test all the [falsey](https://developer.mozilla.org/en-US/docs/Glossary/Falsy) values here.

Reply:
@kitsonk  I think assertExists it's a good idea